src/main/java/org/isf/accounting/service/AccountingIoOperations.java

```diff
@@ -399,4 +399,16 @@ public int compare(Bill o1, Bill o2) {
 		return bills;
 	}
 
+	/**
+	 * Return the entire bill list when the date is between dateFrom and dateTo
+	 *
+	 * @param dateFrom
+	 * @param dateTo
+	 * @return the bill list
+	 * @throws OHServiceException
+	 */
+	public ArrayList<Bill> getAllBills(GregorianCalendar dateFrom, GregorianCalendar dateTo) throws OHServiceException {
+		return (ArrayList<Bill>)billRepository.findByDateBetween(TimeTools.getBeginningOfDay(dateFrom), TimeTools.getBeginningOfNextDay(dateTo));
+	}
+
```

Comment:
Is it not the same as src/main/java/org/isf/accounting/service/AccountingIoOperations.java:291 ?

I think this was the coding history:

1. getBills(GregorianCalendar dateFrom, GregorianCalendar dateTo)
2. getBills(GregorianCalendar dateFrom, GregorianCalendar dateTo, Patient patient)
3. getBills(GregorianCalendar dateFrom, GregorianCalendar dateTo, BillItem billItem)

Methods 2. and 3. should be renamed in a more expressive way, as well as any other "getBills" ones.

Note that refactoring on `core` may affect `gui` as the latter use `manager` packages as endpoints